templates/pages/assets/socket.html.twig

```diff
@@ -45,33 +45,196 @@
       __('Position')
    ) }}
 
+	{% if params["several"] is defined %}
```

Comment:
```several``` form does not seem to work

![image](https://user-images.githubusercontent.com/7335054/173524281-a1d8dc02-0204-4386-8c84-63b6faa108e4.png)

![image](https://user-images.githubusercontent.com/7335054/173524324-d1584733-2bca-4d9e-8526-d79e15bf44ee.png)